lib/mix/lib/mix/tasks/escript.install.ex

```diff
@@ -0,0 +1,113 @@
+defmodule Mix.Tasks.Escript.Install do
+  use Mix.Task
+
+  @shortdoc "Install an escript locally"
+
+  @moduledoc """
+  Install an escript locally.
+
+  If no argument is supplied but there is an escript in the project's root directory
+  (created with `mix escript.build`), then the escript will be installed
+  locally. For example:
+
+      mix do escript.build, escript.install
+
+  The argument is usually a package name and optionally a version:
+
+      mix escript.install phoenix
+      mix escript.install phoenix 0.1.2
+
+  It could also be a remote repository:
+
+      mix escript.install git://example.com/foo.git          # infers Git from the URL
+      mix escript.install https://example.com/foo.git        # infers Git from the URL
+      mix escript.install github.com/example/foo             # fetches git://github.com/example/example.git
+      mix escript.install https://github.com/example/foo     # fetches https://github.com/example/example.git
+      mix escript.install --scm=git example.com/foo          # fetches git://example.com/example
+      mix escript.install --scm=git https://example.com/foo  # fetches https://example.com/example
```

Comment:
We should not infer git from a http url. It limits http urls to the git SCM but a lot of SCMs allow using http as transport.